query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseFunctionField.java

```diff
@@ -129,4 +165,10 @@ protected BaseFunctionField addMessages(Report report) {
         return addArgumentMessages(report.argumentMessages()).
                 addResultMessages(report.resultMessages());
     }
+
+    private String formatErrorCodes(Set<String> errorCodes) {
+        List sortedErrors = new ArrayList<>(errorCodes);
```

Comment:
✏️ Ignoring the odd whitespace issue, a better way to do this would be
```java
        return errorCodes.stream()
                .sorted()
                .collect(Collectors.joining("\r\n"));
```